Makefile

```diff
@@ -29,10 +29,15 @@ endef
 
 all: bin/ekam
 
-bin/ekam: bin/ekam-bootstrap deps
+bin/ekam: bin/ekam-bootstrap | deps
```

Comment:
This changes `deps` to an "order-only dependency": https://www.gnu.org/software/make/manual/html_node/Prerequisite-Types.html. Since it's a phony, it's always out of date, so `bin/ekam` would always be rebuilt even if capnproto had already been cloned.